src/grid.h

```diff
@@ -622,8 +622,17 @@ struct Grid {
         const int font_size = 14 - indent / 2;
         const int grid_border_width =
             cell == doc->rootgrid ? root_grid_spacing : user_grid_outer_spacing - 1;
-        Formatter(r, format, indent,
-                  wxString::Format(L"<grid bordercolor=\"0x%06X\">\n", bordercolor),
+
+        wxString xmlstr(L"<grid");
+        if (bordercolor != 0xA0A0A0) {
+            xmlstr.Append(wxString::Format(wxT(" bordercolor=\"0x%06X\""), bordercolor));
+        }
+        if (user_grid_outer_spacing != 3) {

```

Comment:
`3` is probably a constant somewhere, and if not it should be?

Reply:
Definitely it should be! Thanks for your feedback. Pushed an extra commit for assigning the magic number to a global variable.

Reply:
Thanks for your review and time!